### ISSUE TITLE
Pixelated icon in KDE

### ISSUE THREAD

**Issue by Steinhagen** [lutris/lutris#2188]
The Lutris icon is pixelated when using KDE. This may be as a result of it using the png version of the application instead of the svg one. 
![lutris](https://user-images.githubusercontent.com/4029937/59154401-49c80800-8a7a-11e9-8b1d-56a3143c5aa5.png)

**Comment by natrist** [comment]
This is still an issue as of today. Does anyone have any additional input regarding the issue?

**Comment by xordspar0** [comment]
Is does this always happen with KDE or only with a specific distro?

**Comment by natrist** [comment]
I would be inclined to say it is solely a KDE issue, as I’ve encountered the issue on Arch Linux.

**Comment by Steinhagen** [comment]
> Is does this always happen with KDE or only with a specific distro?
KDE (Plasma more exactly). I've seen this with both Arch and Debian Sid.

**Comment by natrist** [comment]
Just FYI this bug affects the window switcher icon and the panel icon. It might affect other things. What file format are we using to store the image?

**Comment by xordspar0** [comment]
I just installed KDE neon in a VM to try it out and was able to reproduce it.

There are both SVG and PNG images in the PPA. Here's the list:

```
/usr/share/icons/hicolor/16x16/apps/lutris.png
/usr/share/icons/hicolor/22x22/apps/lutris.png
/usr/share/icons/hicolor/24x24/apps/lutris.png
/usr/share/icons/hicolor/32x32/apps/lutris.png
/usr/share/icons/hicolor/48x48/apps/lutris.png
/usr/share/icons/hicolor/scalable/apps/lutris.svg
```

I thought this might be controlled by the .desktop file, but the icon in the application launcher is fine, it's just the window title bar and panel icons that are blurry. I'm not sure what chooses the icon for those.

**Comment by xordspar0** [comment]
This kwin bug thread might indicate that something is going wrong in the interaction between the app, Lutris, and the window manager, kwin.

https://bugs.kde.org/show_bug.cgi?id=351055

**Comment by xordspar0** [comment]
To add more information about the interaction between the X window manager and application, the application, as an X client, is supposed to provide the icon to the window manager via the X protocol. I believe GTK is doing that on Lutris's behalf. You can find out what icons the app is providing to the window manager using the xprop tool. Lutris's X properties seem to be fully filled out, with a wide range of icon sizes to choose from, so I'm not sure what's going on.

<details>
<summary>Results of using xprop on Lutris on KDE neon:</summary>

```
test@test-Standard-PC-Q35-ICH9-2009:~$ xprop
_NET_WM_ICON_GEOMETRY(CARDINAL) = 40, 732, 262, 34
_NET_WM_USER_TIME(CARDINAL) = 19210
_NET_WM_ALLOWED_ACTIONS(ATOM) = _NET_WM_ACTION_MOVE, _NET_WM_ACTION_RESIZE, _NET_WM_ACTION_MINIMIZE, _NET_WM_ACTION_MAXIMIZE_VERT, _NET_WM_ACTION_MAXIMIZE_HORZ, _NET_WM_ACTION_FULLSCREEN, _NET_WM_ACTION_CHANGE_DESKTOP, _NET_WM_ACTION_CLOSE
_NET_WM_DESKTOP(CARDINAL) = 0
_KDE_NET_WM_ACTIVITIES(STRING) = "00000000-0000-0000-0000-000000000000"
WM_STATE(WM_STATE):
                window state: Normal
                icon window: 0x0
_NET_WM_STATE(ATOM) = 
WM_HINTS(WM_HINTS):
                Client accepts input or input focus: True
                Initial state is Normal State.
                bitmap id # to use for icon: 0x2c00038
                bitmap id # of mask for icon: 0x2c0003e
                window id # of group leader: 0x2c00001
_GTK_THEME_VARIANT(UTF8_STRING) = 
XdndAware(ATOM) = BITMAP
_NET_WM_ICON(CARDINAL) =        Icon (22 x 22):
                 ░░░░         
              ░▒▒▒▒▒▒▒▒░░░    
             ▒▒▒▒▒▒▒▒▒▒▒▒▒▒░  
           ░▒▒▒▒▒▒▒▒▒░░░▒▒▒▒▒ 
           ▒▒▒▒▒▒░       ░▒▒▒ 
          ▒▒▒▒▒▒          ░░▒░
         ░▒▒▒▒▒             ░ 
         ▒▒▒▒▒▒░ ░░░░░░     ░ 
         ▒▒▒▒▒▒▒▒░░░░░░░░░    
        ░▒▒▒░ ░▒▒▒░░░░░░░░░░  
        ░▒▒▒░ ░░▒▒░░░░░░░░░░  
        ░▒▒▒  ░░░░░░░░░░░░░░  
        ░▒▒▒  ░░░░░░░░░░░░░░  
         ▒▒▒░ ░░░░░░░░░░░░░░  
         ▒▒▒░ ░░░░░░░░░░░░░░  
         ░▒▒▒  ░░░░░░░░░░░░░  
          ▒▒▒░ ░░░░░░░░░░░░   
           ▒▒▒  ░░░░░░░░░░░   
           ░▒▒▒░ ░░░░░░░░▒    
             ▒▒▒▒░   ░░░▒     
              ░▒▒▒▒▒▒▒▒░      
                 ░░░░         

        Icon (16 x 16):
              ░░░░      
            ▒▒▒▒▒▒▒▒▒░  
           ▒▒▒▒▒▒░░░▒▒▒ 
          ▒▒▒▒░      ▒▒░
         ▒▒▒▒░         ░
         ▒▒▒▒░░░░░░    ░
        ░▒▒▒▒▒▒░░░░░░░  
        ░▒▒  ░▒░░░░░░░░ 
        ░▒▒ ░░░░░░░░░░░ 
        ░▒▒ ░░░░░░░░░░░ 
         ▒▒  ░░░░░░░░░░ 
         ▒▒▒ ░░░░░░░░░  
          ▒▒  ░░░░░░░░  
           ▒▒░ ░░░░░░   
            ▒▒▒░░░░▒    
              ░░░░      

        Icon (24 x 24):
                   ░░           
               ░▒▒▒▒▒▒▒▒░░      
             ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░  
            ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░ 
           ░▒▒▒▒▒▒▒░       ▒▒▒▒ 
          ░▒▒▒▒▒▒░         ▒▒▒▒ 
          ▒▒▒▒▒▒             ░▒ 
         ░▒▒▒▒▒▒    ░         ▒░
         ▒▒▒▒▒▒▒▒░░░░░░░░░    ░ 
         ▒▒▒▒▒▒▒▒▒░░░░░░░░░░░   
        ░▒▒▒▒  ░▒▒▒░░░░░░░░░░░  
        ░▒▒▒▒  ░░▒▒░░░░░░░░░░░  
        ░▒▒▒░  ░░░░░░░░░░░░░░░  
        ░▒▒▒░  ░░░░░░░░░░░░░░░  
         ▒▒▒░  ░░░░░░░░░░░░░░░  
         ▒▒▒▒  ░░░░░░░░░░░░░░░  
         ░▒▒▒  ░░░░░░░░░░░░░░░  
          ▒▒▒▒  ░░░░░░░░░░░░░   
          ░▒▒▒  ░░░░░░░░░░░░░   
           ░▒▒▒  ░░░░░░░░░░░░   
            ░▒▒▒░  ░░░░░░░▒▒    
             ░▒▒▒▒░   ░░░▒░     
               ░▒▒▒▒▒▒▒▒░       
                  ░░░░          

        Icon (32 x 32):
                                        
                   ░▒▒▒▒▒▒▒░░           
                ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░     
               ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░   
              ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒  
             ▒▒▒▒▒▒▒▒▒▒▒▒░░    ░░▒▒▒▒▒░ 
            ▒▒▒▒▒▒▒▒▒▒░          ░▒▒▒▒░ 
           ▒▒▒▒▒▒▒▒▒░            ░▓▒▒▒▒ 
          ░▒▒▒▒▒▒▒▒                 ░▒▒ 
          ▒▒▒▒▒▒▒▒░                  ░▒ 
          ▒▒▒▒▒▒▒▒▒   ░░░░░░░        ▒▓ 
         ░▒▒▒▒▒▒▒▒▒▒░░░░░░░░░░░░        
         ▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░░░░░░░░░     
         ▒▒▒▒▒▒░░░▒▒▒▒░░░░░░░░░░░░░░░   
         ▒▒▒▒▒░   ░▒▒▒▒░░░░░░░░░░░░░░   
         ▒▒▒▒▒   ░░░▒▒▒░░░░░░░░░░░░░░░  
         ▒▒▒▒▒   ░░░░░░░░░░░░░░░░░░░░░  
         ▒▒▒▒▒   ░░░░░░░░░░░░░░░░░░░░░  
         ▒▒▒▒▒   ░░░░░░░░░░░░░░░░░░░░░  
         ▒▒▒▒▒   ░░░░░░░░░░░░░░░░░░░░░  
         ░▒▒▒▒░  ░░░░░░░░░░░░░░░░░░░░   
          ▒▒▒▒▒  ░░░░░░░░░░░░░░░░░░░░   
          ▒▒▒▒▒   ░░░░░░░░░░░░░░░░░░░   
           ▒▒▒▒░  ░░░░░░░░░░░░░░░░░░    
           ▒▒▒▒▒   ░░░░░░░░░░░░░░░░░    
            ▒▒▒▒▒   ░░░░░░░░░░░░░░░░    
             ▒▒▒▒▒   ░░░░░░░░░░░░░▒     
              ▒▒▒▒▒░   ░░░░░░░░░▒▒      
               ▒▒▒▒▒▒░       ░▒▒▒       
                ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒░        
                  ░▒▒▒▒▒▒▒▒▒▒░          
                                        

        Icon (48 x 48):
                                                        
                          ░░░▒▒▒▒░░░░                   
                       ░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒░  ░             
                     ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░        
                   ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░      
                  ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░    
                 ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░   
                ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░  
               ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░          ░▒▒▒▒▒▒▒▒▒  
              ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░               ▒▒▒▒▒▒▒▒  
             ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒                  ▒▓▓▒▒▒▒▒  
            ░▒▒▒▒▒▒▒▒▒▒▒▒▒░                   ▒▓▒▒▒▒▒▒░ 
            ▒▒▒▒▒▒▒▒▒▒▒▒▒                      ░  ░▒▒▒▒ 
           ░▒▒▒▒▒▒▒▒▒▒▒▒                            ░░  
           ▒▒▒▒▒▒▒▒▒▒▒▒▒                            ▒▒░ 
          ░▒▒▒▒▒▒▒▒▒▒▒▒▒      ░░░░░░░░░             ▒▓▒ 
          ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒   ░░░░░░░░░░░░░░          ░▒░ 
          ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░░░░░░░░░░░░░           
         ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░        
         ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░     
         ▒▒▒▒▒▒▒▒▒    ░▒▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░    
         ▒▒▒▒▒▒▒▒▒     ░▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░    
         ▒▒▒▒▒▒▒▒░    ░░░▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░    
         ▒▒▒▒▒▒▒▒░    ░░░░▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░    
         ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░   
         ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░   
         ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░   
         ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░   
         ░▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░   
          ▒▒▒▒▒▒▒░   ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░    
          ▒▒▒▒▒▒▒░   ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░    
          ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░    
           ▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░    
           ▒▒▒▒▒▒▒░   ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░     
           ░▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░     
            ▒▒▒▒▒▒▒░   ░░░░░░░░░░░░░░░░░░░░░░░░░░░░     
             ▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░      
             ░▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░      
              ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░▒       
               ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░▒░       
                ▒▒▒▒▒▒▒▒     ░░░░░░░░░░░░░░░░░▒▒        
                 ▒▒▒▒▒▒▒▒░     ░░░░░░░░░░░░░▒▒▒         
                  ░▒▒▒▒▒▒▒▒░             ░▒▒▒▒          
                   ░▒▒▒▒▒▒▒▒▒▒░░     ░░▒▒▒▒▒░           
                     ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░             
                       ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░               
                          ░░▒▒▒▒▒▒▒▒░░                  
                                                        

        Icon (48 x 48):
                                                        
                          ░░░▒▒▒▒░░░░                   
                       ░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒░  ░             
                     ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░        
                   ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░      
                  ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░    
                 ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░   
                ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░  
               ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░          ░▒▒▒▒▒▒▒▒▒  
              ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░               ▒▒▒▒▒▒▒▒  
             ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒                  ▒▓▓▒▒▒▒▒  
            ░▒▒▒▒▒▒▒▒▒▒▒▒▒░                   ▒▓▒▒▒▒▒▒░ 
            ▒▒▒▒▒▒▒▒▒▒▒▒▒                      ░  ░▒▒▒▒ 
           ░▒▒▒▒▒▒▒▒▒▒▒▒                            ░░  
           ▒▒▒▒▒▒▒▒▒▒▒▒▒                            ▒▒░ 
          ░▒▒▒▒▒▒▒▒▒▒▒▒▒      ░░░░░░░░░             ▒▓▒ 
          ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒   ░░░░░░░░░░░░░░          ░▒░ 
          ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░░░░░░░░░░░░░           
         ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░        
         ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░     
         ▒▒▒▒▒▒▒▒▒    ░▒▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░    
         ▒▒▒▒▒▒▒▒▒     ░▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░    
         ▒▒▒▒▒▒▒▒░    ░░░▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░    
         ▒▒▒▒▒▒▒▒░    ░░░░▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░    
         ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░   
         ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░   
         ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░   
         ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░   
         ░▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░   
          ▒▒▒▒▒▒▒░   ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░    
          ▒▒▒▒▒▒▒░   ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░    
          ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░    
           ▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░    
           ▒▒▒▒▒▒▒░   ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░     
           ░▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░     
            ▒▒▒▒▒▒▒░   ░░░░░░░░░░░░░░░░░░░░░░░░░░░░     
             ▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░░      
             ░▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░░░░      
              ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░░░▒       
               ▒▒▒▒▒▒▒▒    ░░░░░░░░░░░░░░░░░░░░▒░       
                ▒▒▒▒▒▒▒▒     ░░░░░░░░░░░░░░░░░▒▒        
                 ▒▒▒▒▒▒▒▒░     ░░░░░░░░░░░░░▒▒▒         
                  ░▒▒▒▒▒▒▒▒░             ░▒▒▒▒          
                   ░▒▒▒▒▒▒▒▒▒▒░░     ░░▒▒▒▒▒░           
                     ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░             
                       ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░               
                          ░░▒▒▒▒▒▒▒▒░░                  
                                                        


_KDE_NET_WM_USER_CREATION_TIME(CARDINAL) = 11835
_GTK_APP_MENU_OBJECT_PATH(UTF8_STRING) = "/net/lutris/Lutris/menus/appmenu"
_GTK_WINDOW_OBJECT_PATH(UTF8_STRING) = "/net/lutris/Lutris/window/1"
_GTK_APPLICATION_OBJECT_PATH(UTF8_STRING) = "/net/lutris/Lutris"
_GTK_UNIQUE_BUS_NAME(UTF8_STRING) = ":1.43"
_GTK_APPLICATION_ID(UTF8_STRING) = "net.lutris.Lutris"
_NET_WM_OPAQUE_REGION(CARDINAL) = 5, 5, 1012, 671
_MOTIF_WM_HINTS(_MOTIF_WM_HINTS) = 0x2, 0x0, 0x0, 0x0, 0x0
_NET_WM_WINDOW_TYPE(ATOM) = _NET_WM_WINDOW_TYPE_NORMAL
_NET_WM_SYNC_REQUEST_COUNTER(CARDINAL) = 46137355, 46137356
_NET_WM_USER_TIME_WINDOW(WINDOW): window id # 0x2c0000a
WM_CLIENT_LEADER(WINDOW): window id # 0x2c00001
_NET_WM_PID(CARDINAL) = 1249
WM_LOCALE_NAME(STRING) = "en_US.UTF-8"
WM_CLIENT_MACHINE(STRING) = "test-Standard-PC-Q35-ICH9-2009"
WM_NORMAL_HINTS(WM_SIZE_HINTS):
                program specified minimum size: 996 by 87
                program specified base size: 0 by 0
                window gravity: NorthWest
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW, WM_TAKE_FOCUS, _NET_WM_PING, _NET_WM_SYNC_REQUEST
WM_CLASS(STRING) = "lutris", "Lutris"
WM_ICON_NAME(STRING) = "Lutris"
_NET_WM_ICON_NAME(UTF8_STRING) = "Lutris"
WM_NAME(STRING) = "Lutris"
_NET_WM_NAME(UTF8_STRING) = "Lutris"
```
</details>

**Comment by xordspar0** [comment]
I don't think this is a Lutris bug. I can also reproduce this with other GTK apps, such as Gnome Calculator.

![image](https://user-images.githubusercontent.com/2390950/77837097-c1ab1000-712a-11ea-9515-dd8f932bb4cf.png)

**Comment by xordspar0** [comment]
I did some more investigation and learned some new things:

* It's not true that all GTK applications have this problem. It varies from app to app. Gnome Calculator is worse than most because its X properties only provide a single icon that is 48x48.
* X11 only seems to allow clients to provide bitmap images to window managers (as far as I can tell).
* KWin uses multiple icon sizes for different purposes. I took some of Lutris' png images and scaled them to different sizes, then I visually compared my scaled images with the images I saw in Plasma. Note that I'm using the default settings in KDE neon. This might change depending on the user's settings.
  * The title bar icon is rendered at 22x22. Lutris provides a 22x22 image in its X properties. Nevertheless, KWin is taking the 24x24 image and downscaling it, so it doesn't look good. This may be a bug in KWin. I'm trying to track down why this is happening.
  * The task bar icon is rendered at 128x128. KWin is scaling up Lutris' 48x48 icon, which is the largest one that it provides. Lutris should be able to improve this by providing larger icons. For reference, KDE applications provide 16x16, 22x22, 32x32, 48x48, 64x64, and 128x128 icons.
* I have not done any research with window managers other than KWin, or any Wayland compositors, so I don't know how those will work. However, it seems like Gnome on Wayland (what I use personally) is fine.
